packages/remark-parse/lib/tokenize/footnote-definition.js

```diff
@@ -135,6 +135,10 @@ function footnoteDefinition(eat, value, silent) {
       queue += subqueue
       subqueue = ''
 
+      if (queue === lineFeed) {
```

Comment:
I’m doubting if this is the fix 🤔 (see other comments below)

Reply:
When I worked on it I tested it with this and only later added the fixture.

```js
  t.doesNotThrow(function() {
    remark()
      .use({settings: {commonmark: true, footnotes: true}})
      .parse('[^1]: Footnote with trailing newline and space\n ')
  })
```

`node_modules/.bin/tape packages/remark/test.js`